elasticsearch.md

```diff
@@ -143,13 +158,28 @@ service stop elasticsearch
 
 depending on your environment.
 
-#### 2. Uninstall ReadonlyREST
+#### 2. Unpatch ES
+
+If you are using Elasticsearch 8.0.x or newer, you need **an extra pre-uninstallation step**. This will remove all additional files from ROR's installation directory.
```

Comment:
Better say: "this will bring the Elasticsearch installation back to its original state"

As today it only does this copy/delete jar thing. In the future we'd like to do all crazy shit with patch tool.

Reply:
same as above